src/hb-aat-layout-feat-table.hh

```diff
@@ -47,17 +47,18 @@ struct SettingName
   hb_aat_layout_feature_selector_t get_selector () const
   { return (hb_aat_layout_feature_selector_t) (unsigned) setting; }
 
-  void get_info (hb_aat_layout_feature_selector_info_t *s,
-			hb_aat_layout_feature_selector_t default_selector) const
+  hb_aat_layout_feature_selector_info_t get_info (hb_aat_layout_feature_selector_t default_selector) const
```

Comment:
Again, see if can implement as cast operator?

Reply:
It has an argument so I don't will be feasible unlike the other will worked nicely